mkosi/resources/mkosi.md

```diff
@@ -2730,6 +2730,10 @@ Note that the minimum required Python version is 3.9.
   changing ownership to root will work but changing ownership to any other user
   or group will fail.
 
+  Note that chown calls are only suppressed when running package managers, but
+  not during scripts. If this should be required, e.g. for a build script, you
```

Comment:
"during scripts" => "when running scripts", "should be required"' => "is required"